### ISSUE TITLE
Global Overview not showing Zone CP Versions

### ISSUE THREAD

**Issue by matthewhembree** [kumahq/kuma-gui#141]
Multi zone deployment. Post completion of the [multi-zone deployment](https://kuma.io/docs/1.0.7/documentation/deployments/#multi-zone-mode) guide. Everything else seems functional.

Chrome console log:
```
chunk-vendors.js?t=f515544d:47 [Violation] 'requestAnimationFrame' handler took 227ms
chunk-vendors.js?t=f515544d:47 [Violation] 'requestAnimationFrame' handler took 70ms
chunk-vendors.js?t=f515544d:47 [Violation] 'requestAnimationFrame' handler took 65ms
chunk-vendors.js?t=f515544d:47 TypeError: Cannot read property 'toLowerCase' of undefined
    at global-overview.js?t=2c7321c1:1
    at Array.map (<anonymous>)
    at a.dataAdapter (global-overview.js?t=2c7321c1:1)
    at t.apply (chunk-vendors.js?t=f515544d:58)
    at e.get [as data] (chunk-vendors.js?t=f515544d:65)
    at e.validateData (chunk-vendors.js?t=f515544d:65)
    at e.validateData (global-overview.js?t=2c7321c1:1)
    at t.update (chunk-vendors.js?t=f515544d:47)
    at chunk-vendors.js?t=f515544d:47
```

Firefox console log:
```
TypeError: can't access property "toLowerCase", e is undefined
    dataAdapter http://localhost:5681/gui/js/global-overview.js?t=2c7321c1:1
    dataAdapter http://localhost:5681/gui/js/global-overview.js?t=2c7321c1:1
    apply http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:58
    get http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:65
    validateData http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:65
    validateData http://localhost:5681/gui/js/global-overview.js?t=2c7321c1:1
    update http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:47
    requestFrame http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:47
    u http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:47
    requestFrame http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:47
    invalidate http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:47
    invalidate http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:65
    setPropertyValue http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:47
    set http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:47
    validateDataElement http://localhost:5681/gui/js/global-overview.js?t=2c7321c1:1
    validateDataElements http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:65
    validate http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:65
    validate http://localhost:5681/gui/js/global-overview.js?t=2c7321c1:1
    validate http://localhost:5681/gui/js/global-overview.js?t=2c7321c1:1
    _systemUpdate http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:47
    _systemUpdate http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:65
    update http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:47
    u http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:58
    l http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:58
    u http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:58
    update http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:47
    requestFrame http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:47
    u http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:47
    requestFrame http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:47
    invalidate http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:47
    invalidate http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:65
    setPropertyValue http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:47
    set http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:47
    validateDataElement http://localhost:5681/gui/js/global-overview.js?t=2c7321c1:1
    validateDataElements http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:65
    validate http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:65
    validate http://localhost:5681/gui/js/global-overview.js?t=2c7321c1:1
    validate http://localhost:5681/gui/js/global-overview.js?t=2c7321c1:1
    _systemUpdate http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:47
    _systemUpdate http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:65
    update http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:47
    u http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:58
    l http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:58
    u http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:58
    update http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:47
    requestFrame http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:47
    u http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:47
    requestFrame http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:47
    invalidate http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:47
    invalidate http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:65
    setPropertyValue http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:47
    set http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:47
    validateDataElement http://localhost:5681/gui/js/global-overview.js?t=2c7321c1:1
    validateDataElements http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:65
    validate http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:65
    validate http://localhost:5681/gui/js/global-overview.js?t=2c7321c1:1
    validate http://localhost:5681/gui/js/global-overview.js?t=2c7321c1:1
    _systemUpdate http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:47
    _systemUpdate http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:65
    update http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:47
    u http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:58
    l http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:58
    u http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:58
    update http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:47
    requestFrame http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:47
    u http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:47
    requestFrame http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:47
    invalidate http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:47
    invalidate http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:65
    setPropertyValue http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:47
    set http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:47
    validateDataElement http://localhost:5681/gui/js/global-overview.js?t=2c7321c1:1
    validateDataElements http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:65
    validate http://localhost:5681/gui/js/chunk-vendors.js?t=f515544d:65
    validate http://localhost:5681/gui/js/global-overview.js?t=2c7321c1:1
    validate http://localhost:5681/gui/js/global-overview.js?t=2c7321c1:1
chunk-vendors.js:47:129301
```

Screenshot:
![image](https://user-images.githubusercontent.com/47449406/108142675-d74f5e00-708b-11eb-8b11-b7f01a5b5367.png)

Configuration:
```
{
  "apiServer": {
    "auth": {
      "allowFromLocalhost": true,
      "clientCertsDir": ""
    },
    "catalog": {
      "bootstrap": {
        "url": ""
      }
    },
    "corsAllowedDomains": [
      ".*"
    ],
    "http": {
      "enabled": true,
      "interface": "0.0.0.0",
      "port": 5681
    },
    "https": {
      "enabled": true,
      "interface": "0.0.0.0",
      "port": 5682,
      "tlsCertFile": "/var/run/secrets/kuma.io/tls-cert/tls.crt",
      "tlsKeyFile": "/var/run/secrets/kuma.io/tls-cert/tls.key"
    },
    "readOnly": true
  },
  "bootstrapServer": {
    "apiVersion": "v2",
    "params": {
      "adminAccessLogPath": "/dev/null",
      "adminAddress": "127.0.0.1",
      "adminPort": 0,
      "xdsConnectTimeout": "1s",
      "xdsHost": "",
      "xdsPort": 5678
    }
  },
  "defaults": {
    "skipMeshCreation": false
  },
  "diagnostics": {
    "debugEndpoints": false,
    "serverPort": 5680
  },
  "dnsServer": {
    "CIDR": "240.0.0.0/4",
    "domain": "mesh",
    "port": 5653
  },
  "dpServer": {
    "auth": {
      "type": "serviceAccountToken"
    },
    "hds": {
      "checkDefaults": {
        "healthyThreshold": 1,
        "interval": "1s",
        "noTrafficInterval": "1s",
        "timeout": "2s",
        "unhealthyThreshold": 1
      },
      "enabled": false,
      "interval": "5s",
      "refreshInterval": "10s"
    },
    "port": 5678,
    "tlsCertFile": "/var/run/secrets/kuma.io/tls-cert/tls.crt",
    "tlsKeyFile": "/var/run/secrets/kuma.io/tls-cert/tls.key"
  },
  "environment": "kubernetes",
  "general": {
    "dnsCacheTTL": "10s",
    "tlsCertFile": "/var/run/secrets/kuma.io/tls-cert/tls.crt",
    "tlsKeyFile": "/var/run/secrets/kuma.io/tls-cert/tls.key"
  },
  "guiServer": {
    "apiServerUrl": ""
  },
  "metrics": {
    "dataplane": {
      "enabled": true,
      "subscriptionLimit": 10
    },
    "mesh": {
      "maxResyncTimeout": "20s",
      "minResyncTimeout": "1s"
    },
    "zone": {
      "enabled": true,
      "subscriptionLimit": 10
    }
  },
  "mode": "global",
  "monitoringAssignmentServer": {
    "assignmentRefreshInterval": "1s",
    "grpcPort": 5676
  },
  "multizone": {
    "global": {
      "kds": {
        "grpcPort": 5685,
        "refreshInterval": "1s",
        "tlsCertFile": "/var/run/secrets/kuma.io/tls-cert/tls.crt",
        "tlsKeyFile": "/var/run/secrets/kuma.io/tls-cert/tls.key",
        "zoneInsightFlushInterval": "10s"
      },
      "pollTimeout": "500ms"
    },
    "remote": {
      "kds": {
        "refreshInterval": "1s",
        "rootCaFile": ""
      }
    }
  },
  "reports": {
    "enabled": true
  },
  "runtime": {
    "kubernetes": {
      "admissionServer": {
        "address": "",
        "certDir": "/var/run/secrets/kuma.io/tls-cert",
        "port": 5443
      },
      "controlPlaneServiceName": "kuma-control-plane",
      "injector": {
        "caCertFile": "/var/run/secrets/kuma.io/tls-cert/ca.crt",
        "cniEnabled": false,
        "exceptions": {
          "labels": {
            "openshift.io/build.name": "*",
            "openshift.io/deployer-pod-for.name": "*"
          }
        },
        "initContainer": {
          "image": "kong-docker-kuma-docker.bintray.io/kuma-init:1.0.7"
        },
        "sidecarContainer": {
          "adminPort": 9901,
          "drainTime": "30s",
          "gid": 5678,
          "image": "kong-docker-kuma-docker.bintray.io/kuma-dp:1.0.7",
          "livenessProbe": {
            "failureThreshold": 12,
            "initialDelaySeconds": 60,
            "periodSeconds": 5,
            "timeoutSeconds": 3
          },
          "readinessProbe": {
            "failureThreshold": 12,
            "initialDelaySeconds": 1,
            "periodSeconds": 5,
            "successThreshold": 1,
            "timeoutSeconds": 3
          },
          "redirectPortInbound": 15006,
          "redirectPortOutbound": 15001,
          "resources": {
            "limits": {
              "cpu": "1000m",
              "memory": "512Mi"
            },
            "requests": {
              "cpu": "50m",
              "memory": "64Mi"
            }
          },
          "uid": 5678
        },
        "sidecarTraffic": {
          "excludeInboundPorts": [],
          "excludeOutboundPorts": []
        },
        "virtualProbesEnabled": true,
        "virtualProbesPort": 9000
      },
      "marshalingCacheExpirationTime": "5m0s"
    },
    "universal": {
      "dataplaneCleanupAge": "72h0m0s"
    }
  },
  "sdsServer": {
    "dataplaneConfigurationRefreshInterval": "1s"
  },
  "store": {
    "cache": {
      "enabled": true,
      "expirationTime": "1s"
    },
    "kubernetes": {
      "systemNamespace": "kuma-system"
    },
    "postgres": {
      "connectionTimeout": 5,
      "dbName": "kuma",
      "host": "127.0.0.1",
      "maxOpenConnections": 0,
      "maxReconnectInterval": "1m0s",
      "minReconnectInterval": "10s",
      "password": "*****",
      "port": 15432,
      "tls": {
        "caPath": "",
        "certPath": "",
        "keyPath": "",
        "mode": "disable"
      },
      "user": "kuma"
    },
    "type": "kubernetes",
    "upsert": {
      "conflictRetryBaseBackoff": "100ms",
      "conflictRetryMaxTimes": 5
    }
  },
  "xdsServer": {
    "dataplaneConfigurationRefreshInterval": "1s",
    "dataplaneStatusFlushInterval": "10s"
  }
}
```

Thanks!

**Comment by tharun208** [comment]
@bartsmykla, correct me If I am wrong, this same issue got fixed by this pr (https://github.com/kumahq/kuma-gui/pull/139)
right ?

**Comment by matthewhembree** [comment]
This has been resolved in v1.0.8.

![image](https://user-images.githubusercontent.com/47449406/108542968-03e0c100-72aa-11eb-9e24-fb0b4e167abf.png)


Thanks!
